File: r/datastores/blob_storage.R
library(azuremlsdk)

ws <- load_workspace_from_config(path = "../../")
ds_name <- "blob_storage"

ds <- register_azure_blob_container_datastore(
  workspace = ws,
  datastore_name = ds_name,
  container_name = "example",
  account_name = "examplesa",
  account_key = "<add_key>",
  create_if_not_exists = TRUE
)

File: python/datastores/blob_storage.py
from azureml.core import Workspace, Datastore

ws = Workspace.from_config(path="../../")
datastorage_name = "exampleblobstorage"

datastore = Datastore.register_azure_blob_container(
  workspace=ws,
  datastore_name=datastorage_name,
  container_name="example",
  account_name="examplesa",
  account_key="<insert_account_key>",
  create_if_not_exists=True,
)
